src/layers/core/hexagon-layer/hexagon-layer-vertex.glsl

```diff
@@ -22,7 +22,8 @@ uniform float opacity;
 uniform vec3 invisibleColor;
 uniform float radius;
 uniform float angle;
-uniform float enable3d;
+uniform float extruded;
+uniform float radiusScale;
 
 // Lighting constants
 const vec3 ambientColor = vec3(1., 1., 1.);
```

Comment:
> We should migrate all 3d layers to use the same lighting code that PolygonLayer is using.

While I don't have a strong opinion on which lighting implementation we should use. I agree we should have same lighting code, and same support same settings in all the extruded layers. Should be packaged as a shader module, like the lighting code in polygon. 

Finally, this might be longer term but it would be really neat if we could set the lighting settings globally, e.g. through a LightingEffect, instead of having to pass them in as props to every lighting-enabled layer.